.eslintrc.cjs

```diff
@@ -22,4 +22,16 @@ module.exports = {
     'object-shorthand': 'error',
     'prefer-arrow-callback': 'error',
   },
+  overrides: [
```

Comment:
So I needed this to disable some of the warnings that Typescript flags now that `perf/**/*.ts and *.js` are included in the `tsconfig.json` `include` line. I had to include `*.js`, (or at least `runner.js`) in there to get around the below error. In any case, we can reduce this by fixing these errors, but I didn't want to bite that off now; I did fix some `eqeqeq` errors to avoid having to exclude that here too.

```
/Users/nate/dev/roci/replicache/perf/runner.js
  0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: perf/runner.js.
The file must be included in at least one of the projects provided
```

Reply:
I would also be fine adding `.js` files to the main section. It just wasn't needed before.

Reply:
So prior to this PR, files in `perf/` got their ESLint config from `perf/.eslintrc.json`, and *didn't* inherit from the stuff in this file because `root: true` was specified there. Once I remove that and added those files to `tsconfig.json`, all of the "default" rules above *do* apply to `.js` files.

This override section is then relaxing a few of those rules, because the existing `perf/*.js` code violates them wildly. I think we can remove these overridden rules as we migrate JS files in perf to TS and meet the expectations we've set above, but it's ~38 errors right now, and I didn't want to fix them all just to introduce Typescript into the `perf/` directory.

This override also relaxes these rules *just* for JS files in `perf/`, not for any other files, so we keep the same tightness elsewhere we had previously.